M/MSTM/build_tarballs.jl

```diff
@@ -10,14 +10,32 @@ sources = [
 
 # Bash recipe for building across all platforms
 script = raw"""
-cd $WORKSPACE/srcdir/MSTM/code
-mpifort -O2 mpidefs-parallel.f90 mstm-intrinsics.f90 mstm-v4.0.f90 -o "${bindir}/mstm${exeext}"
+if [[ "$target" == x86_64-w64-mingw32 ]]; then
+    cd $WORKSPACE/destdir/include
+    cp $WORKSPACE/destdir/src/mpi.f90 .
+    gfortran -DWIN64 -DINT_PTR_KIND=8 -fno-range-check mpi.f90 || true
```

Comment:
What's the point of this?

Reply:
The Fortran program needs `mpi.mod` to compile, which is not provided by `MicrosoftMPI_jll`, so I have to compile it manually.

Reply:
I think I'm missing what's the point of allowing the failure

Reply:
The compilation will fail due to the lack of the main function, but the `.mod` files will be compiled.

Reply:
Wouldn't `-c` achieve the same goal?

Reply:
Change made!